In [2]:
# Manipulation des données
import pandas as pd
import numpy as np

# Visualisation des données
import matplotlib.pyplot as plt
import seaborn as sns

# Statistiques et analyse
from scipy import stats

In [4]:
data = pd.read_csv("D:/Hackathon_IFRI_IA_G_8/data/raw/Dataset_brute.csv")

In [5]:
df = data.copy()

In [ ]:
df.head()

ID Sexe Age Nationalité Profession (selon catégorie professionnelle)  \
0  1    F  50   Béninoise                                      Artisan   
1  2    M  28   Béninoise                                      Artisan   
2  3    F  69   Béninoise                                          NaN   
3  4    M  68   Béninoise                                 Entrepreneur   
4  5    F  68   Béninoise                                          NaN   

  Adresse (Département) Situation Matrimoniale Motif(s) d'Admission/Asthénie  \
0              Littoral               Marié(e)                             1   
1                   Zou               Marié(e)                             1   
2            Atlantique               Marié(e)                             1   
3               Plateau               Marié(e)                             1   
4                   NaN               Marié(e)                             1   

  Motif(s) d'Admission/Altération de la fonction rénale  \
0                                                  0      
1                                                  1      
2                                                  1      
3                                                  0      
4                                                  1      

  Motif(s) d'Admission/HTA  ... Pathologies/Rétinopathie diabétique  \
0                        0  ...                                   0   
1                        0  ...                                   0   
2                        0  ...                                   0   
3                        1  ...                                   0   
4                        0  ...                                   0   

  Pathologies/Artériosclérose Pathologies/Glaucome Autres Pathologies  \
0                         NaN                    0                NaN   
1                         NaN                    0                NaN   
2                         NaN                    0                NaN   
3                         NaN                    0        Flou visuel   
4                         NaN                    1                NaN   

  Rythme Cardiaque/Sinusal Rythme Cardiaque/Régulier  \
0                        0                       NaN   
1                        0                       NaN   
2                        0                       NaN   
3                        0                       NaN   
4                        0                       Non   

  Trouble de la Repolarisation Stage de l'IRC  \
0                          NaN          CKD 5   
1                          NaN          CKD 5   
2                          NaN          CKD 5   
3                          Oui          CKD 5   
4                          Non          CKD 5   

  Causes Majeure après Diagnostic/HTA Causes Majeure après Diagnostic/Diabète  
0                                   0                                       0  
1                                   0                                       0  
2                                   0                                       0  
3                                   0                                       0  
4                                   1                                       0  

[5 rows x 201 columns]

Voici une explication de ce jeu de données :

1. **Structure des données** :
    - Le jeu de données contient **309 lignes** et **201 colonnes**.
    - Chaque ligne représente un individu ou une observation.
    - Chaque colonne représente une caractéristique ou une variable associée à l'individu.

2. **Colonnes principales** :
    - **ID** : Identifiant unique pour chaque individu.
    - **Sexe** : Sexe de l'individu (M pour masculin, F pour féminin).
    - **Age** : Âge de l'individu.
    - **Nationalité** : Nationalité de l'individu.
    - **Profession (selon catégorie professionnelle)** : Catégorie professionnelle de l'individu.
    - **Adresse (Département)** : Département de résidence de l'individu.
    - **Situation Matrimoniale** : État matrimonial de l'individu (ex. Marié(e)).
    - **Motif(s) d'Admission** : Plusieurs colonnes décrivent les motifs d'admission (ex. Asthénie, HTA, Diabète, etc.).
    - **Personnels Médicaux** : Historique médical personnel (ex. HTA, Diabète, Maladies cardiovasculaires, etc.).
    - **Personnels Chirurgicaux** : Historique chirurgical personnel (ex. Appendicectomie, Césarienne, etc.).
    - **Personnels Familiaux** : Historique médical familial (ex. HTA, Diabète, etc.).
    - **Symptômes** : Symptômes observés (ex. Fièvre, Frissons, Céphalées, etc.).
    - **Tests biologiques** : Résultats de tests biologiques (ex. Créatinine, Protéinurie, Na+, K+, etc.).
    - **Pathologies** : Pathologies diagnostiquées (ex. Rétinopathie hypertensive, Artériosclérose, etc.).
    - **Stage de l'IRC** : Stade de l'insuffisance rénale chronique (ex. CKD 5, CKD 3a, etc.).

3. **Particularités** :
    - Certaines colonnes contiennent des valeurs manquantes (NaN), ce qui indique des données non disponibles pour certains individus.
    - La dernière ligne semble contenir des pourcentages de valeurs manquantes ou des statistiques globales pour chaque colonne.

4. **Utilisation potentielle** :
    - Ce jeu de données peut être utilisé pour analyser les facteurs de risque, les antécédents médicaux, et les diagnostics liés à des pathologies spécifiques.
    - Il peut également servir à construire des modèles prédictifs pour des maladies chroniques comme l'HTA ou le diabète.

Si vous souhaitez une analyse ou une visualisation spécifique, faites-le-moi savoir !